plone/app/versioningbehavior/browser.py

```diff
@@ -104,7 +104,7 @@ def __call__(self):
         if not version_id:
             raise ValueError(u'Missing parameter on the request: version_id')
 
-        field_id = self.request.get('field_id', IPrimaryFieldInfo(self.context).fieldname)
+        field_id = self.request.get('field_id') or IPrimaryFieldInfo(self.context).fieldname
```

Comment:
Sorry, I do not get how this solves it. What do we get with ``IPrimaryFieldInfo(self.context)``? ``None`` or a ``LookupError``? 

In code I would express it like so:

```suggestion
        field_id = self.request.get('field_id')
        if not field_id:
            primary_field_info = IPrimaryFieldInfo(self.context, None)
            if primary_field_info:
                field_id = primary_field_info.fieldname
            else:
               # What now? 
               raise ValueError("Houston, we have no field_id")
```

Reply:
> Sorry, I do not get how this solves it. What do we get with `IPrimaryFieldInfo(self.context)`? `None` or a `LookupError`?

We get `TypeError` (Could not adapt) (#51)

I didn't want to change / break existing behaviors, that's why my change is very minimal ...

Reply:
phew. ok, so thats the idea... so, lets go with it for now.